modules/core/ImagerySystem.js

```diff
@@ -398,6 +400,98 @@ export class ImagerySystem extends AbstractSystem {
   }
 
 
+  /**
```

Comment:
This method is missing a top-line comment here in the comment block. Bryan's changes are filled with extremely helpful comments, so I'd try to use the jsdoc format like he does to capture a brief synopsis of what the method does, when it should be called, what it assumes is true when it's called, and what it returns. 

Writing comments is *very* hard to do!

Reply:
something I need to get a lot better at doing! Bryan does have great comments and I'll try to copy his format